api/db-cdr/generate-cdr/make-bq-denormalized-tables.sh

```diff
@@ -91,8 +91,8 @@ SELECT P.PERSON_ID,
 	c2.CONCEPT_NAME as SOURCE_NAME,
 	c2.CONCEPT_CODE as SOURCE_CODE,
 	c2.VOCABULARY_ID as SOURCE_VOCABULARY,
-	t.VISIT_OCCURRENCE_ID as VISIT_ID,
-	 v.visit_concept_id,
+	CASE WHEN t.VISIT_OCCURRENCE_ID is null then 0 else t.VISIT_OCCURRENCE_ID end as VISIT_ID,
+	CASE WHEN v.visit_concept_id is null then 0 else v.visit_concept_id end as visit_concept_id,
```

Comment:
We may want to find a better way to do this when we have time.

Reply:
Agreed, will have a followup PR that can address this.

Reply:
Sounds good.